index.html

```diff
@@ -54,59 +49,80 @@
 The <dfn>`Promise`</dfn> and <dfn>`DOMException`</dfn> interfaces are defined
 in [[!DOM]].
 
-## <dfn>SerialPort</dfn> interface
+<section data-dfn-for="Serial" data-link-for="Serial">
+<h2><dfn>Serial</dfn> interface</h2>
+
 <pre class="idl">
-[constructor(DOMString path, optional SerialOptions options)]
+[Exposed=Window, SecureContext]
+partial interface Navigator {
+  readonly attribute Serial serial;
+};
+
+[Exposed=DedicatedWorker, SecureContext]
+partial interface WorkerNavigator {
+  readonly attribute Serial serial;
+};
+
+[Exposed=(DedicatedWorker,Window), SecureContext]
+interface Serial : EventTarget {
```

Comment:
The alternative is to make `getPorts()` and `requestPorts()` static methods on `SerialPort` and move `onconnect` and `ondisconnect` to something like `onserialportconnected` and `onserialportdisconnected` events on the `document` object similar to the Gamepad (dis)connect events.

I prefer keeping everything under a single "root" object for the API because it avoids naming conflicts with other specifications and also helps make the API compatible with non-browser contexts like Node.js where you would do something like,

```js
const serial = require('serial');
const port = serial.requestPort(...);
```

I'm not worried about developers being confused by `Serial` vs. `SerialPort` since neither of them is constructible so those names won't appear in code anyways.